### ISSUE TITLE
vc4: Add support for glInvalidateFramebuffer()

### ISSUE THREAD

**Issue by anholt** [anholt/mesa#20]
This would let apps tell us that we can avoid storing the depth/stencil contents, saving memory bandwidth.

**Comment by loganmc10** [comment]
I'm having issues using glDiscardFramebufferEXT (GLES2), would this be the same issue?

If I try to #define GL_GLEXT_PROTOTYPES it tells me it can't find the symbol, and if I try to declare the function using the symbol (PFNGLDISCARDFRAMEBUFFEREXTPROC) it segfaults.

**Comment by anholt** [comment]
No, it's that you can't directly use extensions in GL on Linux.  You need to use glXGetProcAddress() / eglGetProcAddress() to get the function pointer.  I'd recommend using libepoxy to avoid having to deal with that.

**Comment by loganmc10** [comment]
Ok thanks, i guess the closed source driver works differently than most, using that driver i could just include gl2ext.h and then use glDiscardFramebufferEXT directly

**Comment by anholt** [comment]
Yeah, lots of implementations accidentally expose their functions like this.  It's a shame that Khronos has standards saying that GL on Linux shouldn't expose functions like a normal library.
